test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8665,8 +8641,13 @@ def setup_rnn_tests():
                 ('lstm', 'lstm', {}),
                 ('gru', 'gru', {})
         ):
+            # Need Add between list of tensors
+            script_test_min_opset_version = 11
+            # These models fail to compile in scripting mode. This should be fixed.
+            if base == 'elman' or base == 'lstm':
```

Comment:
Thanks for the fix. Can you also mention the failure for elman and lstm in the comment?